api_tests/applications/views/test_application_list.py

```diff
@@ -48,69 +49,67 @@ def setUp(self):
                 }
             }
         }
+        return sample_data
 
-    def test_user1_should_see_only_their_applications(self):
-        res = self.app.get(self.user1_list_url, auth=self.user1.auth)
-        assert_equal(len(res.json['data']),
-                     len(self.user1_apps))
+    def test_user_should_see_only_their_applications(self, app, user, user_app, user_list_url):
+        res = app.get(user_list_url, auth=user.auth)
+        assert len(res.json['data']) == ApiOAuth2Application.objects.count()
```

Comment:
This assertion isn't consistent with the test method name. The length of the returned data should be equal to the length of the user's application list, not the count of all applications.